students/elaine_x/session03/list_lab_ex.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+'''
+##########################
+#Python 210
+#Session 03 - List Lab
+#Elaine Xu
+#Jan 28,2019
+###########################
+'''
+
+############# Series 1 ################################
+print("Series 1\nList of Fruit:")
+FRUITS = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(FRUITS)
+
+#Ask the user for another fruit and add it to the end of the list
+FRUIT = input('Enter another fruit: ')
+FRUITS.append(FRUIT)
+print(FRUITS)
+
+#Ask user for a number, display the number and the fruit corresponding to the number
+NUMBER = int(input('Enter an index number (1 is first): '))
+print("The "+ str(NUMBER) +"th fruit is "+FRUITS[NUMBER-1])
+
+#Add another fruit to the beginning of the list using “+” and display the list
+print()
+print('Adding "Kiwi" to the beginning of the list')
+FRUITS = ['Kiwi'] + FRUITS
+print(FRUITS)
+
+#Add another fruit to the beginning of the list using insert() and display the list
+print()
+print('Adding "Banana" to the beginning of the list')
+FRUITS.insert(0, 'Banana')
+print(FRUITS)
+
+#Display all the fruits that begin with “P”, using a for loop
+print()
+print('Display all the fruits that begin with "P"')
+for item in FRUITS:
+    if item[0] == "P":
```

Comment:
`if item.lower().startswith("p"):`  If they want it to be case sensitive, drop the `.lower()`